packages/components/src/components/files/FileAttachmentForm.tsx

```diff
@@ -34,6 +34,7 @@ interface FileAttachmentFormProps {
     cancelText?: string
     initialFileNames?: Array<string>
     initialFiles?: {[key:string]: File}
+    index?: any
```

Comment:
Should this type be "number"? or what is an example of what a usage will pass in as this property?

Reply:
Yup, 'number' is better--fixed!